spec/puppet-lint/plugins/check_classes/inherits_across_namespaces_spec.rb

```diff
@@ -6,27 +6,27 @@
   context 'class inheriting from parent in same module namespace' do
     let(:code) { 'class foo::bar inherits foo { }' }
 
-    it 'should not detect any problems' do
+    it 'does not detect any problems' do
       expect(problems).to have(0).problems
     end
   end
 
   context 'class inheriting from sister in same module namespace' do
     let(:code) { 'class foo::bar inherits foo::baz { }' }
 
-    it 'should not detect any problems' do
+    it 'does not detect any problems' do
       expect(problems).to have(0).problems
     end
   end
 
   context 'class inheriting from another module namespace' do
     let(:code) { 'class foo::bar inherits baz { }' }
 
-    it 'should only detect a single problem' do
+    it 'onlies detect a single problem' do
```

Comment:
there are still many occurrences of "onlies" - 99 of them I think... would it be an issue to fix all of them?

Reply:
🤣